server/src/services/storage.service.ts

```diff
@@ -63,6 +68,38 @@ export class StorageService extends BaseService {
   }
 
   @OnJob({ name: JobName.DELETE_FILES, queue: QueueName.BACKGROUND_TASK })
```

Comment:
`createReadmeFile` absolutely shouldn't run on `DELETE_FILES` lol

Reply:
That should be on the next function down instead.

Reply:
Job queueing tests are quite limited still, yea :(